src/library_glfw.js

```diff
@@ -419,7 +419,7 @@ var LibraryGLFW = {
         GLFW.active.height = GLFW.active.storedHeight;
```

Comment:
the code above this can also be changed - no need for an `event` parameter, and no need for all the `removeEventListener`. probably the rest can be removed too.